<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWindow</class>
 <widget class="QWidget" name="ConnectionWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../resources/icons/buttons/buttonicons.qrc">
    <normaloff>:/icons/buttons/connect.png</normaloff>:/icons/buttons/connect.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>USB</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="selectedUSBportComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="usbDisconnectPushButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/icons/buttons/buttonicons.qrc">
          <normaloff>:/icons/buttons/disconnect.png</normaloff>:/icons/buttons/disconnect.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="usbConnectPushButton">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/icons/buttons/buttonicons.qrc">
          <normaloff>:/icons/buttons/server_connect.png</normaloff>:/icons/buttons/server_connect.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Database</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="hostnameLineEdit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="databaseNameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hostname:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Database name:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="usernameLineEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="databaseDisconnectPushButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/icons/buttons/buttonicons.qrc">
          <normaloff>:/icons/buttons/disconnect.png</normaloff>:/icons/buttons/disconnect.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="databaseConnectPushButton">
        <property name="text">
         <string>Connect</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../resources/icons/buttons/buttonicons.qrc">
          <normaloff>:/icons/buttons/database_connect.png</normaloff>:/icons/buttons/database_connect.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Database driver:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="databaseDriverComboBox">
        <item>
         <property name="text">
          <string>QMYSQL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MARIADB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QODBC</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/icons/buttons/buttonicons.qrc"/>
  <include location="../../../../resources/icons/buttons/buttonicons.qrc"/>
  <include location="../../../../resources/icons/buttons/buttonicons.qrc"/>
 </resources>
 <connections/>
</ui>
